security-command-center/snippets/src/main/java/vtwo/source/UpdateFindingSource.java

```diff
@@ -0,0 +1,92 @@
+/*
+ * Copyright 2024 Google LLC
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ * http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+package vtwo.source;
+
+import com.google.cloud.securitycenter.v2.Finding;
+import com.google.cloud.securitycenter.v2.FindingName;
+import com.google.cloud.securitycenter.v2.SecurityCenterClient;
+import com.google.cloud.securitycenter.v2.UpdateFindingRequest;
+import com.google.protobuf.FieldMask;
+import com.google.protobuf.Timestamp;
+import com.google.protobuf.Value;
+import java.io.IOException;
+import java.time.Instant;
+
+public class UpdateFindingSource {
+
+  public static void main(String[] args) throws IOException {
+    // TODO: Replace the below variables.
+    // organizationId: Google Cloud Organization id.
+    String organizationId = "{google-cloud-organization-id}";
+
+    // Specify the location to list the findings.
+    String location = "global";
+
+    // Specify the source-id.
+    String sourceId = "{source-id}";
+
+    // Specify the finding-id.
+    String findingId = "{finding-id}";
+
+    updateFinding(organizationId, location, sourceId, findingId);
+  }
+
+  // Creates or updates a finding.
+  public static Finding updateFinding(String organizationId,
+      String location, String sourceId, String findingId) {
+    try (SecurityCenterClient client = SecurityCenterClient.create()) {
+      // Optionally FindingName or String can be used.
```

Comment:
maybe change this to `Instead of using the FindingName, a plain String can also be used. E.g.:`.
WDYT?

Reply:
done